### PR TITLE
Civ 6011

### PR DESCRIPTION
**Pull request by pliao-hmcts**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-6010
https://tools.hmcts.net/jira/browse/CIV-6011

### Change description ###
allow listing officer access GA "Hearing Scheduled" event and enter its screen 1
screen shots are in both ticket

import ga ccd
image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ga-ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEventGAspec-GAR3CP.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "CaseEventID": "HEARING_SCHEDULED",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "hearing-schedule-access"
```
Comment (pliao-hmcts) [inline ga-ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEventGAspec-GAR3CP.json:8]: need system update

File: ga-ccd-definition/AuthorisationCaseState/AuthorisationCaseStateGAspec-GAR3CP-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "CaseStateID": "LISTING_FOR_A_HEARING",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "hearing-schedule-access"
```
Comment (karthick-mohanasundaram-HMCTS) [inline ga-ccd-definition/AuthorisationCaseState/AuthorisationCaseStateGAspec-GAR3CP-nonprod.json:8]: I believe systemupdate userrole is missing. could you verify please?

File: ga-ccd-definition/CaseField/CaseFieldGAspec-GAR3CP-nonprod.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "ID": "hearingNoticeApplicationDetail",
+    "Label": "Application details",
+    "FieldType": "FixedList",
+    "FieldTypeParameter": "ApplicationDetailsOption",
+    "SecurityClassification": "Public"
+  },
+  {
+  "CaseTypeID": "GENERALAPPLICATION",
```
Comment (karthick-mohanasundaram-HMCTS) [inline ga-ccd-definition/CaseField/CaseFieldGAspec-GAR3CP-nonprod.json:11]: could you fix the indentation?

**Comment by karthick-mohanasundaram-HMCTS** [comment]
@pliao-hmcts could you please attach the screen shot of importing the ga ccd?

- [ ] screenshot

File: ga-ccd-definition/AuthorisationCaseState/AuthorisationCaseStateGAspec-GAR3CP-nonprod.json (hunk 2 above)
Comment (pliao-hmcts) [reply ga-ccd-definition/AuthorisationCaseState/AuthorisationCaseStateGAspec-GAR3CP-nonprod.json:8]: systemupdate userrole is in AuthorisationCaseStateGAspec.json

**Comment by pliao-hmcts** [comment]
refactor in 6502

**Pull request closed without merging**